### PR TITLE
Add e2e tests

### PR DESCRIPTION
**Pull request by ollyrowe**

### CODE REVIEW

File: cypress/e2e/hard-mode.cy.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+  it("prevents user from dragging letters onto disabled tiles", () => {
+    const solutionLetters = getTodaysSolutionLetters();
+
+    const firstLetter = solutionLetters[0].letter;
+
+    // Loop through each tile on the board
+    for (let row = 1; row <= 6; row++) {
+      for (let column = 1; column <= 5; column++) {
+        // If this tile doesn't for part of the solution
```
Comment (emilielake) [inline cypress/e2e/hard-mode.cy.ts:49]: *form

File: cypress/e2e/spell-word.cy.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      cy.getTileFromRack(letter).placeOnBoard(location);
+    });
+
+    letters.forEach(({ location }) => {
+      // Each letter should be green
+      cy.getTileFromBoard(location).shouldHaveGreenBackground();
+    });
+  });
+
+  it("displays an incorrect as grey", () => {
```
Comment (emilielake) [inline cypress/e2e/spell-word.cy.ts:21]: word

**Review by emilielake — CHANGES_REQUESTED** [review]
See comments

File: cypress/e2e/yesterdays-game.cy.ts [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          cy.getByTestID("yesterday-modal")
+            .findTile({ row, column })
+            .shouldHaveGreyBackground();
+        } else {
+          // Otherwise the tile placeholder should be disabled
+          cy.getByTestID("yesterday-modal")
+            .findPlaceholder({ column, row })
+            .shouldBeDisabledPlaceholder();
+        }
+      }
```
[the remark this answers is not in the document]
Comment (ollyrowe) [reply cypress/e2e/yesterdays-game.cy.ts:32]: Creating new issue

**Review by emilielake — APPROVED** [review]

**Pull request merged**